change/@fluentui-react-spinbutton-2e315b0b-23fc-4c22-a235-d2a5cc90cc22.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 260 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26849&commitId=423682054686c4ed945d17d62dd7c6a9a75ecf2a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1588 screenshots unchanged, 0 screenshots with different dimensions and 260 screenshots with visible difference.


  <details><summary><b>unknown</b> 260 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.SpinButton-error.default.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|4739| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|5045| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|5026| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|5051| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|4758| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|4718| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|5051| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|4924| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|5187| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|5187| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|5091| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|5014| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|5020| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|5114| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|54| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|56| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|55| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|47| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|76| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|81| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|47| Changed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|56| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|57| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|57| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|48| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|77| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|83| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|48| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|4370| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|4657| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|4659| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|4657| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|4390| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|4379| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|4657| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|5044| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|5211| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|5211| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|5217| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|5038| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|5038| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|5217| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|53| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|48| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|48| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|46| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|71| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|73| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|46| Changed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|55| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|50| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|50| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|48| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|73| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|75| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|48| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|4102| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|4256| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|4256| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|4246| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|4104| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|4104| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|4241| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|4924| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|5187| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|5187| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|5091| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|5014| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|5020| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|5114| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|73| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|67| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|67| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|65| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|91| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|92| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|62| Changed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|75| Changed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|69| Changed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|69| Changed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|67| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|93| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|94| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|64| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|4226| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|4379| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|4379| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|4366| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|4228| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|4228| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|4366| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|5048| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|5330| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|5338| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|5213| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|5157| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|5174| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|5215| Changed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|54| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|52| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|52| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|50| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|75| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|78| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|50| Changed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|56| Changed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|54| Changed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|54| Changed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|52| Changed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|77| Changed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|80| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|51| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|4096| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|4250| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|4240| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|4240| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|4098| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|4095| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|4235| Changed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|4918| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|5180| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|5084| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|5084| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|5008| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|4918| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|5108| Changed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|67| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|61| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|59| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|59| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|84| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|67| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|56| Changed|
|SpinButton Converged.At Max Bound.focused.chromium.png|69| Changed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|63| Changed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|61| Changed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|61| Changed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|86| Changed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|69| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|58| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|4096| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|4240| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|4250| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|4240| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|4095| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|4098| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|4235| Changed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|4918| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|5084| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|5180| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|5084| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|4918| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|5014| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|5108| Changed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|67| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|59| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|61| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|59| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|67| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|85| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|56| Changed|
|SpinButton Converged.At Min Bound.focused.chromium.png|69| Changed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|61| Changed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|63| Changed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|61| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|69| Changed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|87| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|58| Changed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|18| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|18| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|94| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|94| Changed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|37| Changed|
|SpinButton Converged.Disabled.focused.chromium.png|37| Changed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|37| Changed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|37| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|37| Changed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|3992| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|4147| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|4147| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|4136| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|3994| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|3994| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|4132| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|4794| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|5056| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|5056| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|4960| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|4883| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|4889| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|4983| Changed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|168| Changed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|162| Changed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|162| Changed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|160| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|186| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|187| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|158| Changed|
|SpinButton Converged.Display Value.focused.chromium.png|180| Changed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|174| Changed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|174| Changed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|172| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|197| Changed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|198| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|169| Changed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|56| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|99| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|98| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|92| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|77| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|83| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|92| Changed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|55| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|93| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|93| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|91| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|73| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|75| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|91| Changed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|75| Changed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|95| Changed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|95| Changed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|93| Changed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|93| Changed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|94| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|93| Changed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|56| Changed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|54| Changed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|54| Changed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|51| Changed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|77| Changed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|80| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|51| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|62| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|64| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|3085| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|3730| Changed|
|SpinButton Converged.Size- small - RTL.default.chromium.png|92| Changed|
|SpinButton Converged.Size- small.default.chromium.png|95| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|51| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>